src/ec2/face_detector.py

```
@@ -14,33 +15,50 @@ def __init__(self, output_dir="detected_faces", tolerance=0.6):
+        return False
 
     def process_frame(self, frame):
+        self.frame_skip_counter += 1
+        if self.frame_skip_counter % 20 != 0:
+            print('skip')
+            return []
+
+        else:
+            filename = f"frame_{self.frame_skip_counter}.jpg"
+            filepath = os.path.join('saved_frames', filename)
+            success = cv2.imwrite(filepath, frame)
```

Ruff (0.12.2)

Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)